src/components/Button/Button.js

```diff
@@ -17,6 +17,8 @@ const Button = (props) => {
     buttonOuterClassName,
```

Comment:
🔧 can't comment on the right line 😅 but `onClickHandler` is called and errors in browser. Because of the new href functionality it's not actually needed here so best to default it to `onClickHandler = {() => {}}`